### PR TITLE
Remove requirement for JWT when running suspend_users.js script

### PR DESCRIPTION
**Pull request by kprist**
Archived decks will be listed to have been archived by user with id: -1 (the system)

Also add helper scripts to mass importing deckids owned by users to be suspended. See README

### CODE REVIEW

File: application/queue/deckids_to_userids.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* global db */
```
Comment (TBoonX) [inline application/queue/deckids_to_userids.js:1]: If the db object is global, then how (from where) should this file be called?

Comment (kprist) [reply application/queue/deckids_to_userids.js:1]: This is a mongo shell script file, can only run under the mongo shell. This file is not used directly, but via the `application/queue/deckids_import.sh` script.

File: application/queue/suspend_users.js [hunk 2]
```diff
@@ -13,10 +11,15 @@ const config = require('../configuration'),
 
 let deckidsToUserids = {};
 
-if (process.argv[2] === undefined) {
-  console.log('Please provide a JWT!');
-  return;
-}
+// generate a `isReviewer` jwt the deck service will accept  
+const jwt = require('jsonwebtoken');
+const adminToken = jwt.sign({
```
Comment (TBoonX) [inline application/queue/suspend_users.js]: Please use controllers/jwt.js for creating tokens.

**Review by TBoonX — APPROVED** [review]
When jwt.js is used its ready to be merged!

File: application/queue/suspend_users.js (hunk 2 above)
Comment (kprist) [reply application/queue/suspend_users.js]: I am generating a "dummy" token that has the necessary credentials. The `controllers/jwt.js` file (which, by the way is not a controller really, should be in `lib` or something) includes a method that requires a user object. Or do you mean, just move this code in the other file ?

Comment (TBoonX) [reply application/queue/suspend_users.js]: Right, jwt.js in a lib folder would fit better.
You are using a user object in which the email attribute is missing, which is no problem. Using jwt.createToken() with your object should reduce duplicated code.

**Comment by kadevgraaf** [comment]
@TBoonX - do you want to do one last check before merging? 
@bwulff  - does this help you in (automatically/batch-ily) running the user suspending / spam removal script?

**Comment by TBoonX** [comment]
Ready to be merged from my side!

**Comment by kadevgraaf** [comment]
Thanks! 👍

**Pull request merged**